benchmark/misc/freelist.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
 
 var common = require('../common.js');
-var FreeList = require('internal/freelist').FreeList;
 
 var bench = common.createBenchmark(main, {
   n: [100000]
 });
 
 function main(conf) {
+  const FreeList = require('internal/freelist').FreeList;
```

Comment:
`require` was moved there, so it will work with adding `-expose_internals` only in `common.js`. It was the result of discusion here: https://github.com/nodejs/node/pull/8547#pullrequestreview-202699